tests/framework/installer/ceph_manifests.go

```diff
@@ -391,7 +391,7 @@ spec:
                   type: integer
                   minimum: 1
                   maximum: 65535
-                securePort:
+                securePort: 443
```

Comment:
Actually we won't want the port specified here. This is just the schema.

Reply:
Ah, that's the CRD. Foolishness on my part.